MantidQt/CustomInterfaces/src/Muon/MuonAnalysis.cpp

```diff
@@ -318,6 +318,18 @@ void MuonAnalysis::initLayout() {
   // Manage User Directories
   connect(m_uiForm.manageDirectoriesBtn, SIGNAL(clicked()), this,
           SLOT(openDirectoryDialog()));
+  // m_uiForm.
```

Comment:
thought I had caught all of them. Its an old bit of code, not needed anymore (as you have seen). 

Thank you for spotting it